client/src/app/+my-library/my-videos/my-videos.component.ts

```diff
@@ -92,7 +97,7 @@ export class MyVideosComponent implements OnInit, DisableForReuseHook {
   getVideosObservable (page: number, sort: VideoSortField) {
     const newPagination = immutableAssign(this.pagination, { currentPage: page })
 
-    return this.videoService.getMyVideos(newPagination, sort, this.videosSearch)
+    return this.videoService.getMyVideos(newPagination, this.sort, this.videosSearch)
```

Comment:
What about the `sort` parameter?

Reply:
It doesn't get updated in my experience, unfortunately.

Reply:
Because you don't update the URL when you sort videos

Reply:
I tried updating the URL, no to avail. It seems the scope gets frozen when `.bind(this)` is called, which means, with the component initialization.

Reply:
I think there is no way using the frozen `sort` value passed at `.bind` time can do here.